static/js/containers/admin/ChannelModerationPage.js

```diff
@@ -155,6 +159,6 @@ export default R.compose(
   connect(mapStateToProps),
   withPostModeration,
   withCommentModeration,
-  withSingleColumn("edit-channel"),
-  withSpinnerLoading
+  withChannelHeader(false),
+  withSingleColumn("edit-channel")
```

Comment:
not sure how much we are concerned about this, but this change removes the loading spinner from the moderation page, so when you navigate to it it says "No outstanding reports" for a second before the request to get moderation reports finishes, and then it flashes to show the reports. Before I think there was a loading spinner.

Reply:
I added back a spinner, what do you think of the refactor here?